### PR TITLE
Feature : Ability to hide page in side menu

### PR DESCRIPTION
**Pull request by tomheno**
This feature add an option to hide a page in side-menu.

### CODE REVIEW

File: src/Pages/Page.php [hunk 1]
```diff
@@ -58,6 +60,11 @@ public static function getNavigationSort()
         return static::$navigationSort;
     }
 
+    public static function getDisplayInMenu()
```
Comment (ryangjchandler) [inline src/Pages/Page.php:63]:
```suggestion
    public static function shouldDisplayInMenu()
```

File: src/Pages/Page.php [hunk 2]
```diff
@@ -84,7 +91,7 @@ public static function getPageTitle()
 
     public static function navigationItems()
     {
-        return [
+            return static::getDisplayInMenu() === true ? [
```
Comment (ryangjchandler) [inline src/Pages/Page.php:94]:
```suggestion
        return static::shouldDisplayInMenu() ? [
```

**Review by ryangjchandler — CHANGES_REQUESTED** [review]

**Comment by ryangjchandler** [comment]
@tomheno I've made a few code style changes - I like the idea behind this feature but it can already be achieved in user-land by overwriting the `navigationItems` method. Will see what @danharrin thinks too.

**Review by ryangjchandler — APPROVED** [review]

**Comment by tomheno** [comment]
@ryangjchandler Thanks for your feedbacks. Awaiting for @danharrin pov then :)

**Comment by danharrin** [comment]
Gonna leave this one, as it really is as simple as overriding the `navigationItems()` method with `return [];`. Feel free to PR laravel-filament/docs with this though 👍

**Comment by tomheno** [comment]
Thanks, PR made : https://github.com/laravel-filament/docs/pull/2

**Pull request closed without merging**